### DISCUSSION TITLE
Update the package to use with latest api 202309.

### DISCUSSION THREAD

**Discussion by Dev-Waheed** [EcomPHP/tiktokshop-php#20] [Ideas]
Hello,

I appreciate your work on this package; it has been quite helpful. I'm curious about your plans for maintaining it and whether you intend to update the package to support the latest TikTok Shop API version 202309. Currently, it seems to be compatible with the older version 202212. I'm looking forward to your response. Thank you.

**Comment by nVuln** [comment]
202309 is in development, I'll make a PR for this version in 1-2 days

**Reply by nVuln** [reply]
I'll update my work at #21

**Reply by Dev-Waheed** [reply]
Thank you for the update and your commitment to maintaining the package. I appreciate your efforts in keeping it compatible with the latest TikTok Shop API version.
